pymc_marketing/mmm/plot.py

```diff
@@ -570,7 +570,7 @@ def plot_curve(
     subplot_kwargs : dict, optional
         Addtional kwargs to while creating the fig and axes
     sample_kwargs : dict, optional
-        Kwargs for the :func:`plot_curve` function
+        Kwargs for the :func:`plot_sample` function
```

Comment:
I think is `plot_samples` right? (s at the end)?

Reply:
Okay! just done this.